lingui.config.js

```diff
@@ -0,0 +1,15 @@
+/** @type {import('@lingui/conf').LinguiConfig} */
+module.exports = {
+  locales: ['en'],
+  extractBabelOptions: {
+    presets: ['@babel/preset-typescript', '@babel/preset-react'],
+  },
+  sourceLocale: 'en',
```

Comment:
We should source these from the `next.config.js` file.

```suggestion
  locales: nextConfig.i18n.locales,
  extractBabelOptions: {
    presets: ['@babel/preset-typescript', '@babel/preset-react'],
  },
  sourceLocale: nextConfig.i18n.defaultLocale,
```

Reply:
Totally agree, in a lot of projects we add a `locales.js` file in the root of the project that is then used by `next.config.js` and a potential Sanity setup. This file could then be used for `lingui.config.js` as well :)

Reply:
Ah, yeah. I've implemented this now! @maeertin